iphone/Maps/UI/PlacePage/Components/OpenInAppActionSheet/OpenInApplication.swift

```diff
@@ -0,0 +1,114 @@
+enum OpenInApplication: Int, CaseIterable {
+  case osm
+  case googleMaps
+  case appleMaps
+  case osmAnd
+  case yandexMaps
+  case dGis
+  case cityMapper
+  case moovit
+  case uber
+  case waze
+}
+
+extension OpenInApplication {
+
+  static var allCases: [OpenInApplication] {
+    [
+      .osm,
+      .googleMaps,
+      .appleMaps,
+      .osmAnd,
+      .yandexMaps,
+      .dGis,
+      .cityMapper,
+      .moovit, 
+      .uber,
+      .waze
+    ]
+  }
+
+  static var availableApps: [OpenInApplication] {
+    // OSM should always be first in the list.
+    let sortedApps: [OpenInApplication] = [.osm] + allCases.filter { $0 != .osm }.sorted(by: { $0.name < $1.name })
+    return sortedApps.filter { UIApplication.shared.canOpenURL(URL(string: $0.scheme)!) }
+  }
+
+  var name: String {
+    switch self {
+    case .osm:
+      return "OpenStreetMap"
+    case .googleMaps:
+      return "Google Maps"
+    case .appleMaps:
+      return "Apple Maps"
+    case .osmAnd:
+      return "OsmAnd Maps"
```

Comment:
OsmAnd

Reply:
But this app is actually called `OsmAnd Maps` in iOS

![image](https://github.com/organicmaps/organicmaps/assets/79797627/1f1bf35c-b89c-4730-94e8-9d1610fcf5c3)